packages/trader/src/AppV2/Components/ContractCard/contract-cards-sections.tsx

```diff
@@ -32,10 +34,11 @@ const ContractCardsSections = ({ onScroll, positions }: TContractCardsSections)
                             return sellTime && formatDate({ time: sellTime }) === date;
                         })}
                     />
+                    {isLoadingMore && <Loading is_fullscreen={false} />}
                 </div>
             ))}
         </div>
     );
 };
 
-export default ContractCardsSections;
+export default React.memo(ContractCardsSections);
```

Comment:
memoizing the component is essential to prevent all the items from re-rendering on every next batch fetching which scrolling down in Closed tab.